src/ADP.Portal.Api/Models/Ado/AdoGroup.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Newtonsoft.Json;
```

Comment:
Why are we still using newtonsoft instead of System.Text.Json?

Reply:
Removed the unnecessary code